docs/sources/configure-client/go-span-profiles.md

```diff
@@ -7,38 +7,52 @@ weight: 100
 
 # Span profiles with Traces to profiles for Go
 
```

Comment:
The blog post about span profiles has some great content, so I used a few of the paragraphs for the introduction and linked to the blog post.

Blog post: https://grafana.com/blog/2024/02/06/combining-tracing-and-profiling-for-enhanced-observability-introducing-span-profiles/